### ISSUE TITLE
Logging doesn't always capture the buffer?

### ISSUE THREAD

**Issue by Rilauven** [talvo/potato#471]
I'm trying to figure out what I'm doing wrong, I have a tendency to forget to start logging until after a scene has started. The default setting is Include Buffer From: Main Window. I always leave it like this, but it seems to be a roll of the dice if it will actually save the buffer to the log or just begin a new log at the current line.

I'm reasonably sure this isn't a bug and I'm doing something wrong, Any help explaining how logging works would be greatly appreciated! I'm using the 64 bit Windows version. 2.0.0b17

**Comment by Aranis** [comment]
Bumping this, as I've seen the same behavior. I believe it to be a bug.

Steps to reproduce:
- Connect to any MU*.
- Click on "Logging."
- Click on "Show Log Window."
- Do not change any options.
- Click on the ellipsis.
- Choose a file name (I've tested a variety of names, the behavior is always the same).
- Click "Log."

When you open the file, there is no buffer from the main window included in the log. I tried version potato-2.0.0b16-win32, and it didn't have this behavior.
